### PR TITLE
added change to tasks file and all tests

### PR DESCRIPTION
**Pull request by femalves**

### CODE REVIEW

**Comment by femalves** [comment]
https://github.com/actions/setup-python/issues/672 
"Breaking changes
Hello everyone. The Python 2.7.x will be removed from [python-versions](https://github.com/actions/python-versions) and it won't be possible to set up Python 2.7.x by setup-python.

The motivation for the changes
Python2.7 is not [supported](https://www.python.org/doc/sunset-python-2/) since January 1, 2020."

**Comment by tjacovich** [comment]
To update the tests to `python3`
This section in the `python-actions.yaml` [file](https://github.com/adsabs/ADSOrcid/blob/dc50c78e29f58e31064c52b265fc8c82a6b2bdfe/.github/workflows/python_actions.yml#L12C1-L14C28)
```
  steps:
    - uses: actions/checkout@v2
    - uses: actions/setup-python@v2
      with:
        python-version: 2.7
```

Should become:
```
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-python@v2
        with:
          python-version: "3.8"
```

**Review by tjacovich — CHANGES_REQUESTED** [review]
Overall looks good. Just a couple of formatting things and the update to the github action.

File: requirements.txt [hunk 1]
```diff
@@ -3,6 +3,6 @@ alembic==0.9.1
 cachetools==1.1.5
 python-Levenshtein==0.12.0
 psycopg2==2.8.6
-numpy==1.16.6
-matplotlib==2.0.2
+numpy==1.22 # 1.16.6 is not compatible with M1. 
+matplotlib==3.5.0 #2.0.2 is not compatible with Python 3.8
```
[the remark this answers is not in the document]
Comment (femalves) [reply requirements.txt:7]: Yup. Thanks for the reminder. Done!

**Review by tjacovich — COMMENTED** [review]
This looks good to me. Just one incredibly small potential tweak in `tasks.py` and then one potential test tweak. It looks like one of the assert methods used in the tests for `models.py` has been deprecated. Would you be willing to incorporate that fix into this PR?

One of the offending lines (it looks like there might be two of them in total):
```
ADSOrcid/tests/test_models.py::Test::test_models
  /home/runner/work/ADSOrcid/ADSOrcid/ADSOrcid/tests/test_models.py:61: DeprecationWarning: Please use assertRaisesRegex instead.
    with self.assertRaisesRegexp(Exception, 'IntegrityError'):
```

**Review by tjacovich — APPROVED** [review]

**Pull request merged**